doc/Calibration.md

```diff
@@ -82,6 +82,20 @@ You can assess the tolerance using either an M6 Allen key or the printed hexagon
 ![image](./images/OrcaToleranceTes_m6.jpg)  
 ![image](./images/OrcaToleranceTest_print.jpg)  
 
+# Advanced Calibration
+
+## Max Volumetric speed
+This is a test designed to calibrate the maximum volumetric speed of the specific filament. The generic or 3rd party filament types may not have the correct volumetric flow rate set in the filament. This test will help you to find the maximum volumetric speed of the filament.
+
+Click on "Calibration" -> "More" -> "Max Volumetric Speed". You will be promted to enter the settings for the test: start volumetric speed, end volumentric speed, and step. It is recommended to use the default values, unless you already have an idea of the lower or upper limit for your filament. Select "OK", slice the plate, and send it to the printer. 
+
+Once printed, take note of where the layers begin to fail. Using calipers or a ruler, measure the height of the print at that point. Then, return to OrcaSlicer and in the "Preview" tab, make sure the color scheme "flow" is selected. Scroll down to the layer height that you measured, and click on the toolhead slider. This will indicate the max flow level for your filmanet. Save this to your filament profile.

```

Comment:
While it's convenient to check the flow in preview, we can also calculate the value directly: start + height*step

Reply:
Read my comment again, I realised my previous comment was confusing. 

We should also mention that `we can also calculate the value directly: start + height*step`

Reply:
@SoftFever My apologies, I wasn’t receiving GitHub notifications so didn’t see where this ended up. Taking a new photo for the docs today and will update properly

Reply:
@SoftFever Pushed some changes up. Let me know what you think